developer_guide.md

```diff
@@ -4,8 +4,8 @@
 Create a symbolic link inside your GOPATH to the location you checked out the code
 
 ```sh
-mkdir -p ${GOPATH}/src/github.com/jlewi
-ln -sf ${GIT_TRAINING} ${GOPATH}/src/mlkube.io
+mkdir -p ${GOPATH}/src/github.com/tensorflow
+ln -sf ${GIT_TRAINING} ${GOPATH}/src/github.com/tensorflow/k8s
```

Comment:
It seems like `${GIT_TRAINING}` is the same as `${GOPATH}/src/github.com/tensorflow/k8s` in some case.

Reply:
I'm not sure I understand the question. Its certainly possible to check out the repository directly into your GOLANG src tree in which case you don't need to create a symbolic link.